app/views/products/edit.html.haml

```diff
@@ -0,0 +1,26 @@
+%h1 Edit product
```

Comment:
is editing and deleting products in scope for this feature?

Reply:
We are only using the edit product part to verify that our admin functionality is working properly (ie. a regular user should not be able to see the edit btn).

Reply:
Well you can make a button then, not add the whole edit functionality. I think you have a whole story in ur PT board that is about this functionality.

Reply:
Changes made accordingly.